components/infobox/commons/infobox.lua

```diff
@@ -54,7 +54,7 @@ end
 ---@param injector WidgetInjector?
 ---@return self
 function Infobox:widgetInjector(injector)
-	self.injector = injector
+	self.injector = injector or self.injector
```

Comment:
What's the purpose of this? There shouldn't be any calls to this without an injector, no?

Reply:
there are in all the bases, e.g.
https://github.com/Liquipedia/Lua-Modules/blob/a0501da4084ce5321c6e207a804ba6986557bcf4/components/infobox/commons/infobox_league.lua#L257